src/components/TitleCardList.tsx

```diff
@@ -152,52 +40,169 @@ const TitleCardList: React.FC<IProps> = ({
 }) => (
-  <Table className={className}>
-    <TableHeader>
-      {icon && <StyledIcon name={icon} />}
+  <Box
+    bg="background"
+    boxShadow="tableBox"
+    width="100%"
+    margin={8}
+    border-radius="sm"
+    className={className}
+  >
+    <Flex
+      bg="ednBackground"
+      align="center"
+      justifyContent="space-between"
+      padding={4}
+      flexDirection="row-reverse"
+      fontWeight="600"
+      borderBottom={0.25}
+      borderBottomStyle="solid"
+      borderBottomColor="text"
+    >
+      {icon && (
+        <ChakraIcon
+          as={Icon}
+          name={icon}
+          _hover={{
+            path: { fill: "tranparent" },
+          }}
+        />
+      )}
       <Translation id={headerKey} />
       {isCode && (
-        <CodeBoxHeader>
-          <Red />
-          <Yellow />
-          <Green />
-        </CodeBoxHeader>
+        <Hide below="s">
+          <Flex>
+            <Box
+              width="12px"
+              height="12px"
+              bg="fail300"
+              marginRight={2}
+              borderRadius="full"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="gridYellow"
+              marginRight={2}
+              borderRadius="full"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="success300"
+              marginRight={2}
+              borderRadius="full"
+            />
+          </Flex>
+        </Hide>
       )}
-    </TableHeader>
+    </Flex>
     {content.map((listItem, idx) => {
       const { title, description, caption, link, image, alt, id } = listItem
       const isLink = !!link
+
       return isLink ? (
-        <ItemLink key={id || idx} to={link}>
-          {image && <Image image={image} alt={alt || ""} />}
-          <LeftContainer>
-            <ItemTitle>{title}</ItemTitle>
+        <LinkBox
+          key={id || idx}
+          as={Link}
+          href={link}
+          display="flex"
+          textDecoration="none"
+          hideArrow
+          justifyContent="space-between"
+          color="text"
+          marginBottom="1px"
+          padding={4}
+          width="100%"
+          boxShadow="0px 1px 1px var(--eth-colors-tableItemBoxShadow)"
+          _hover={{
+            textDecoration: "none",
+            boxShadow: "0 0 1px var(--eth-colors-primary)",
+            bg: "primary100",
+            color: "black",
+          }}
+        >
+          {image && (
+            <Box
+              as={GatsbyImage}
+              marginTop={1}
+              marginRight={4}
+              minWidth={5}
+              image={image}
+              alt={alt || ""}
+            />
+          )}
+          <Flex flex="1 1 75%" flexDirection="column" marginRight={8}>
+            <Box>{title}</Box>
 
-            <ItemDesc>{description}</ItemDesc>
-          </LeftContainer>
+            <Box fontSize="s" marginBottom="0" opacity="0.7">
```

Comment:
(Once `LinkOverlay` is added)

Due to CSS stacking context, `opacity` will not work for any instance here on any element that has this which is wrapping either the `description` or `caption` content. Because the use of `LinkOverlay` allows for the whole box to be clickable, opacity applied to any text will be above the clickable area, and therefore make that part of the clickable area ineffective.

I suggest the following:

Before the component return:
```ts
// This will be accessible with color contrast
const descriptionCaptionColor = useColorModeValue(
  "blackAlpha.700",
  "whiteAlpha.700"
)
```

And then for each instance of the `opacity` prop (also the `fontSize` token needs to be updated):
```suggestion
            <Box fontSize="sm" marginBottom="0" color={descriptionCaptionColor}>
```